src/tools/genpolicy/src/registry.rs

```diff
@@ -374,6 +374,7 @@ pub fn add_verity_to_store(cache_file: &str, diff_id: &str, verity_hash: &str) -
     let mut writer = BufWriter::new(&file);
     writeln!(writer, "{}", serialized)?;
     writer.flush()?;
+    #[allow(unstable_name_collisions)]
```

Comment:
Any idea why my Agent build worked without this change?

Reply:
I tried to build genpolicy as well. So, this error cropped up when building genpolicy.

Reply:
Ah, for some reason I thought this was Agent code, not genpolicy. Got it!